packages/plans-grid/src/plans-interval-toggle/index.tsx

```diff
@@ -57,20 +57,24 @@ const PlansIntervalToggle: React.FunctionComponent< PlansIntervalToggleProps > =
 	maxMonthlyDiscountPercentage,
 	className = '',
 } ) => {
-	const { __, hasTranslation } = useI18n();
+	const { __, _x, hasTranslation } = useI18n();
 	const locale = useLocale();
 
 	const fallbackMonthlyLabel = __( 'Pay monthly', __i18n_text_domain__ );
 	// Translators: intended as "pay monthly", as opposed to "pay annually"
-	const newMonthlyLabel = __( 'Monthly', __i18n_text_domain__ );
+	const newMonthlyLabel = _x( 'Monthly', 'Adverb (as in "Pay monthly")', __i18n_text_domain__ );
 	const monthlyLabel =
-		locale === 'en' || hasTranslation?.( 'Monthly' ) ? newMonthlyLabel : fallbackMonthlyLabel;
+		locale === 'en' || hasTranslation?.( 'Monthly', 'Adverb (as in "Pay monthly")' )
```

Comment:
Hey @yuliyan , just double-checking with you if this is the correct way of using `hasTranslation` with `context` — thank you!

Reply:
Yes, that's correct. Note that after migrating [@wordpress/react-i18n](https://github.com/Automattic/wp-calypso/pull/51406), `hasTranslation` will also accept `domain` as the third argument, but since it has a fallback value, there shouldn't be needed any changes here.